inc/vcf/summary_report_writer.hpp

```diff
@@ -110,23 +110,31 @@ namespace ebi
     class SummaryReportWriter : public ReportWriter
     {
       public:
-        SummaryReportWriter(std::ostream &out) : out(out) {}
+        SummaryReportWriter(std::string filename)
+        {
+            fout.open(filename, std::ios::app);
```

Comment:
to append might be useful, but everyone will get some kind of error because everyone will assume that the report will be overwritten. I can almost hear this rippling from the future:
> Why it shows the first same error as before? if I just fixed it! :rage:

Reply:
I just saw below that the `main()` ensures that the file doesn't exist. Still, I would use just `std::ios::out`